client/get_deployment_filter.go

```
@@ -0,0 +1,32 @@
+package client
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+)
+
+//CommandOptionsGetDeploymentFilter used for the command options filter (flags)
+type CommandOptionsGetDeploymentFilter struct {
+	Filter string
+}
+
+//GetDeploymentFilter return the deployment from the client
+func GetDeploymentFilter(cli *Cli, commandOptions *CommandOptionsGetDeploymentFilter) Deployments {
+
+	//Build URL
+	resturl := fmt.Sprintf("resources/deployments/filter?filters=")
+	resturl += url.QueryEscape(commandOptions.Filter)
+
+	// fmt.Println(resturl)
+	// os.Exit(0)
+
+	//Call rest client
+	deployments := Deployments{}
+	if err := cli.Client.DoRequest(http.MethodGet, resturl, nil, &deployments); err != nil {
+		log.Fatal("Error rest call: ", err)

```

log.Fatal calls os.exit(1), I think it is better to return the error here.